test/core/bitswap.spec.js

```diff
@@ -129,7 +129,10 @@ describe('bitswap', function () {
     })
   })
 
-  describe('transfer a block between', () => {
+  // TODO does not pass in browser because `createServer is not a function`
+  // https://github.com/libp2p/js-libp2p-websockets/issues/70
```

Comment:
Wait, is this a new issue? These tests have been passing fine forever.

Reply:
Mystery solved, the issue was caused by this non necessary change at https://github.com/ipfs/js-ipfs/pull/1103#discussion_r168933535

Reply:
Could you remove this comments?